src/main/java/com/insightfullogic/honest_profiler/core/parser/Method.java

```diff
@@ -40,10 +40,25 @@ public Method(long methodId, String fileName, String className, String methodNam
         this.methodName = methodName;
     }
 
+    // Avoid formatting class name in copy()
+    private Method(long methodId,
+                   String fileName,
+                   String className,
+                   String methodName,
+                   boolean dummy)
+    {
+        this.methodId = methodId;
+        this.fileName = fileName;
+        this.className = className;
+        this.methodName = methodName;
+    }
+
```

Comment:
Thanks for helping with review this PR @nitsanw and @chhsiao90 - greatly appreciated! I still think we should keep to one build system and not have it as part of this PR. I don't mind if we move to gradle in future, but let's not bundle that up in these changes.